pkg/telemetry/telemetry.go

```diff
@@ -137,6 +139,13 @@ func GetTelemetryClient(ctx context.Context) TelemetryClient {
 // SetCobraCommandContext sets the telemetry values for the command being executed.
 func (e *CLIAnalyticsEventMetadata) SetCobraCommandContext(cmd *cobra.Command) {
 	e.CommandPath = cmd.CommandPath()
+	var flags []string
+	cmd.Flags().VisitAll(func(f *pflag.Flag) {
```

Comment:
Will it be triggered even if wrong flags are passed? Is it gonna run after or before an error is thrown?

Reply:
The telemetry events are sent inside the `PersistentPreRunE`, which run after the flag parsing.
So no, no telemetry will be sent if any flags are wrong / missing.